frontend/src/scenes/billing/BillingEnrollment.tsx

```diff
@@ -0,0 +1,69 @@
+import { Button, Card, Col, Row, Skeleton, Spin } from 'antd'
+import { useActions, useValues } from 'kea'
+import React from 'react'
+import { PlanInterface } from '~/types'
+import { billingLogic } from './billingLogic'
+import defaultImg from 'public/plan-default.svg'
+
+function Plan({ plan, onSubscribe }: { plan: PlanInterface; onSubscribe: (plan: PlanInterface) => void }): JSX.Element {
+    return (
+        <Card>
+            <div className="cursor-pointer" onClick={() => onSubscribe(plan)}>
+                <img src={plan.image_url || defaultImg} alt="" height={100} width={100} />
+                <h3 style={{ fontSize: 22 }}>{plan.name}</h3>
+            </div>
+            <div>
+                <Button
+                    data-attr="btn-subscribe-now"
+                    data-plan={plan.key}
+                    type="primary"
+                    onClick={() => onSubscribe(plan)}
+                >
+                    Subscribe now
+                </Button>
+            </div>
+            <div className="plan-description" dangerouslySetInnerHTML={{ __html: plan.description }} />
```

Comment:
Could we do it without providing HTML from the database?

Reply:
Addressed, seehttps://github.com/PostHog/posthog-production/pull/69 for details. Now fetching HTML from a static template in the `posthog-production` repo.